neuraxle/steps/util.py

```diff
@@ -352,17 +356,15 @@ def save(self, pipeline: 'Pipeline', data_container: DataContainer) -> 'Pipeline
     def load(self, pipeline: 'Pipeline', data_container: DataContainer) -> 'Pipeline':
         return pipeline
 
-
-class BaseValueCachingWrapper(MetaStepMixin, BaseStep, ABC):
+class ValueCachingWrapper(MetaStepMixin, BaseStep):
     """
-    Base class for a value caching wrapper that enables value caching after a fit.
+    Value caching wrapper wraps a step that inherits OutputTransformerMixin,
```

Comment:
It doesnt seem that it inherits from OutputTransformerMixin. And why would it do so? 

I think that a OutputValueCachingWrapper would be a new and different class which would also save/reload Y and not just X. To be done in a different PR, this one would get too huge (open issue perhaps).